samples/xamarin/ToDoItems/ToDoItems.Core/Services/CosmosDBService.cs

```diff
@@ -74,7 +79,13 @@ public async static Task<List<ToDoItem>> GetToDoItems()
 
             return todos;
         }
+        // </GetToDoItems>
 
+        
+        // <GetCompletedToDoItems>        
+        /// <summary> 
```

Comment:
remove summary and returns. 
This would apply on all places where we are encasing on ``// <codeTag>` for documentation linking